PushSignedBinariesIntoBuild.cmd

```diff
@@ -22,24 +22,34 @@ set BinaryOutputDirectory=%BinaryOutputDirectory%\%Platform%_%Configuration%\
 set SigningDirectory=%BinaryOutputDirectory%\..\Signing
 
 :: Copy all multitargeted assemblies to their locations
-call :CopyFilesForMultitargeting Json.Schema.dll            || goto :ExitFailed
-call :CopyFilesForMultitargeting Json.Pointer.dll           || goto :ExitFailed
-call :CopyFilesForMultitargeting Json.Schema.ToDotNet.dll   || goto :ExitFailed
-call :CopyFilesForMultitargeting Json.Schema.Validation.dll || goto :ExitFailed
+call :CopyDllForMultitargeting Json.Schema.dll            || goto :ExitFailed
```

Comment:
>CopyDllForMultitargeting [](start = 6, length = 24)

I've never liked this "ForMultitargeting" part of the name; it took me a while to figure out what it meant. How about `:CopySignedLibraryToBuildOutputDirectory`. (Note I say "Library" rather than "Dll" because the "Exe" can also be a .dll; see my other naming suggestion below.

Reply:
To clarify: I still don't know what that portion of the name _means_, but now at least I know what the code _does_.

---
In reply to: [213832995](https://github.com/Microsoft/jschema/pull/30#discussion_r213832995) [](ancestors = 213832995)